mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -89,7 +89,6 @@ def _parse_ann_info(self, img_info, ann_info):
         Args:
             ann_info (list[dict]): Annotation info of an image.
             with_mask (bool): Whether to parse mask annotations.
-
```

Comment:
Please follow [this](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/models/sot/stark.py#L81-L90) to change the format of docstring.
This PR miss a log of blank lines in docstring.